src/activity_item_handling.cpp

```diff
@@ -488,11 +488,9 @@ static void move_items( const tripoint &src, bool from_vehicle,
         indices.pop_back();
         quantities.pop_back();
 
-        item *temp_item = nullptr;
-
-        temp_item = ( from_vehicle == true ) ?
-                    g->m.item_from( s_veh, s_cargo, index ) :
-                    g->m.item_from( source, index );
+        item *temp_item = from_vehicle ?
+                            g->m.item_from( s_veh, s_cargo, index ) :
+                            g->m.item_from( source, index );
```

Comment:
The indentation here makes astyle complain.

Reply:
Astyled.